test-case-reporting/app.ts

```diff
@@ -128,6 +128,38 @@ app.get('/test-results/build/:buildNumber', async (req, res) => {
   }
 });
 
+app.get('/test-cases/stats/:stat', async (req, res) => {
+  const {stat} = req.params;
+  const {json, count = 10} = req.query;
+
+  try {
+    const sampleSize = Number(count);
+
+    if (Number.isNaN(sampleSize)) {
+      throw new TypeError('sampleSize is not a number');
```

Comment:
```suggestion
      throw new TypeError(`Expected a number for parameter 'count'; got ${count}`);
```

Reply:
Done. I'll try to make my error messages more informative in the future :+1: